src/helpers/isValidTokenValue.ts

```diff
@@ -0,0 +1,14 @@
+import BigNumber from 'bignumber.js';
+
+import { LOW_LIQUIDITY_DISPLAY_TRESHOLD } from 'appConstants';
+import { TokenType } from 'types';
+
+export const isValidTokenValue = (token: TokenType) => {
```

Comment:
I'd use a hook and memoize this computation if this is used in an iteration

Reply:
used mostly as a helper function to show/hide components, when used on expensive iterations, those are already memoized